### ISSUE TITLE
Link of a localized string isn't shown (Even in the source language)

### ISSUE THREAD

**Issue by rpkoller** [WebDevStudios/custom-post-type-ui#221]
If you take a look at string ID number 137 ( i am talking about the 1.0.1 branch i've downloaded yesterday)

```
See <a href=\"http://codex.wordpress.org/Function_Reference/register_post_type#Parameters\">Available options</a> in the \"menu_position\" section. Range of 5-100
```
"Available options" isn't shown as a link.

![bildschirmfoto 2015-02-11 um 15 20 09 2](https://cloud.githubusercontent.com/assets/1665422/6148409/93aae718-b201-11e4-9a77-67e4c08ba748.png)

Aside availble should be lowercase?

**Comment by tw2113** [comment]
Ooops. That's my own fault once I added strip_tags() to the get_p() method I created.

I'll fix that at a different point release.

**Comment by rpkoller** [comment]
ah oki doki. but the string won't change in the pot file? So i could leave it as is and just localize?

**Comment by tw2113** [comment]
The tag is being stripped after it's already been passed through the internationalization/localization functions. So if you're leaving the link tag in the po/mo files, then it'll be linked once I fix the strip tag aspect, however which way I manage that.

**Comment by tw2113** [comment]
Will be fixed once 7094f49ec5649ee28b6750e25cece5df41a5ad09 gets pushed up to GH
